src/Tools/dotnet-monitor/DiagnosticsMonitorCommandHandler.cs

```diff
@@ -313,12 +316,19 @@ private static void ConfigureMetricsEndpoint(IConfigurationBuilder builder, bool
             {
                 {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.Endpoints)), string.Join(';', metricEndpoints)},
                 {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.Enabled)), enableMetrics.ToString()},
-                {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.UpdateIntervalSeconds)), MetricsOptionsDefaults.UpdateIntervalSeconds.ToString()},
                 {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.MetricCount)), MetricsOptionsDefaults.MetricCount.ToString()},
                 {ConfigurationPath.Combine(ConfigurationKeys.Metrics, nameof(MetricsOptions.IncludeDefaultProviders)), MetricsOptionsDefaults.IncludeDefaultProviders.ToString()}
             });
         }
 
+        private static void ConfigureGlobalMetrics(IConfigurationBuilder builder)
```

Comment:
This isn't really necessary because we are using the extension method you've added for the IntervalSeconds property.

Reply:
I would still rather keep this. For example dotnet-monitor config show will actually produce a value here

Reply:
I actually don't know if I like that behavior though. I think it's also inconsistent as to what has default values as in-memory collections.